### ISSUE TITLE
Naming review- MONDO:0011827 patent ductus arteriosus

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#9870]
I see MONDO:0011827 patent ductus arteriosus includes the OMIM Phenotypic series and Orphanet's Familial patent arterial duct .
Would it be more appropriate for this term to indicate the "familial" aspect of the PDA concept here? Currently it's only in the 'related' synonyms.  It would help disambiguate with the HPO term for the phenotype https://hpo.jax.org/browse/term/HP:0001643 for MedGen mapping. But I also understand the importance of honor the sources' preferred names, so will do our best to reflect records here. 
Right now UMLS has the Orphanet concept "Familial patent arterial duct" on a CUI with this SNOMED concept- 1179298002.  And the HPO term is on the PDA concept (C0013274).

**Comment by sabrinatoro** [comment]
Thank you @kanems. 
I added "familial patent ductus arteriosus" as an exact synonym, and updated the existing synonyms to make them "exact". 
Please let me know if updating the synonyms is not working for you. As you said, we are trying to honor the sources' preferred names; however, the need to clearly distinguish the Mondo term from the HPO term at the term label level would be a good argument to update the name.
